Pradeo/pradeo/tests/match_threats.json

```
@@ -0,0 +1,42 @@
+{
+    "input": {
+        "sekoiaio": {
+            "intake": {
+                "dialect": "pradeo",
+                "dialect_uuid": "e30f7bcc-7c55-4666-9d32-61a0aa75a2c3"
+            }
+        },
+        "message": "Nov 15 08:51:42 subdomain.pradeo.net mtd-pradeosecuritysystems[6030]: {\"USER\":\"test_user\",\"SECTION\":\"MTD\\/Apps\",\"ACTION\":\"app_checking\",\"DESCRIPTION\":\"\",\"ITEM\":{\"APPLICATION_ID\":\"55936212\",\"APP_PKG_NAME\":\"com.an_app\",\"APP_VERSION\":\"4.394.10003\",\"APP_SHA1_SIG\":\"a92675ab3dafb37399c47a75ceac8effc4cb401d\"},\"ACTION_VALUES\":{\"ALLOWED\":\"true\",\"ACTION\":\"automatic\",\"POLICY\":\"Green\",\"MATCH_THREATS\":[\"cat_phone_cache_send\",\"cat_phone_device_info_send\",\"cat_phone_hardware_send\",\"cat_user_contact_info_send\",\"match_encrypt_with_key_downloaded_from_network\",\"match_exec_command_downloaded_from_network\",\"match_hide_app_icon_from_launcher\",\"match_priority\",\"match_rootkit\",\"match_rootkit_warning\",\"match_sms\"]}}\n"
+    },
+    "expected": {
+        "package": {
+            "checksum": "a92675ab3dafb37399c47a75ceac8effc4cb401d",
+            "name": "com.an_app",
+            "version": "4.394.10003"
+        },
+        "action": {
+            "name": "app_checking",
```

```suggestion
        "event": {
            "reason": "app_checking",
```

This looks more like `event.reason` to me.

---

The value comes from `"ACTION"` field. I think it is more the name rather than the reason